src/main.rs

```diff
@@ -13,35 +12,30 @@
 
  Copyright (c) 2018, Bryan D. Ashby
  See LICENSE.TXT
-
 */
+
 extern crate clap;
 extern crate codepage_437;
 extern crate docopt;
-#[macro_use] extern crate serde_derive;
-#[cfg(windows)] extern crate winapi;
+#[macro_use]
+extern crate serde_derive;
+#[cfg(windows)]
+extern crate winapi;
 
-use codepage_437::CP437_CONTROL;
-use codepage_437::FromCp437;
-use codepage_437::IntoCp437;
 use clap::crate_version;
+use codepage_437::{FromCp437, IntoCp437, CP437_CONTROL};
 use docopt::Docopt;
 use std::fs;
-#[cfg(windows)] use std::io::Error;
+use std::io::Error;
+use std::io::ErrorKind;
+use std::io::Write;
 use std::net::TcpStream;
-#[cfg(not (windows))]use std::os::unix::io::AsRawFd;
-#[cfg(windows)] use std::os::windows::io::AsRawSocket;
-#[cfg(windows)] use std::os::windows::raw::HANDLE;
 use std::path::Path;
 use std::process;
 use std::process::Command;
 use std::vec::Vec;
-#[cfg(windows)] use winapi::shared::minwindef::TRUE;
-#[cfg(windows)] use winapi::um::handleapi::DuplicateHandle;
-#[cfg(windows)] use winapi::um::processthreadsapi::GetCurrentProcess;
-#[cfg(windows)] use winapi::um::winnt::DUPLICATE_SAME_ACCESS;
 
-const USAGE: &'static str = "
+const USAGE: &str = "
```

Comment:
Can you explain this change? My understanding was `const foo: &'static str = "...";` is how one wants to declare a immutable const string.

Reply:
It is.  It's just that, in this case, the `'static` lifetime is the default and you don't need to write it explicitly. This change is, more than anything, aesthetic.